works.html

```diff
@@ -71,7 +71,300 @@
+                  Центре «Астана» (МФЦА) и осуществляет деятельность, основанную
+                  на нормах и принципах Английского общего права.
+                </p>
+                <p class="main__wrapper-common-text-paragrph">
+                  МФЦА – это уникальный центр, который использует лучшие практики
+                  и подходы международных финансовых центров и сотрудничает с
+                  крупными корпорациями и небольшими компаниями, недавно
+                  появившимися на рынке.
+                </p>
+                <p class="main__wrapper-common-text-paragrph">
+                  – это уникальный центр, который использует лучшие практики и
+                  подходы международных финансовых центров и сотрудничает с
+                  крупными корпорациями и небольшими компаниями, недавно
+                  появившимися на рынке.
+                </p>
+              </div>
+            </div>
+            <div class="main__wrapper-minerals">
+              <h4 class="main__wrapper-minerals-title">Мinerals Operating</h4>
+              <p class="main__wrapper-common-text-paragrph">
+                Это молодая компания, которая оказывает услуги в сфере
+                недропользования на территории Казахстана. Мы одна из первых
+                компаний являющихся лицензированным консультантом в сфере
+                минеральных ресурсов (лицензия №AFSA-A-LA-2020-0004). Мinerals
+                Operating зарегистрирована в Международном Финансовом Центре
+                «Астана» (МФЦА) и осуществляет деятельность, основанную на нормах
+                и принципах Английского общего права. Мinerals Operating – это
+                молодая компания, которая оказывает услуги в сфере
+                недропользования на территории Казахстана. Мы одна из первых
+                компаний являющихся лицензированным консультантом в сфере
+                минеральных ресурсов (лицензия №AFSA-A-LA-2020-0004). Мinerals
+                Operating зарегистрирована в Международном Финансовом Центре
+                «Астана» (МФЦА) и осуществляет деятельность, основанную на нормах
+                и принципах Английского общего права.
+              </p>
+              <p class="main__wrapper-common-text-paragrph">
+                Мinerals Operating – это молодая компания, которая оказывает
+                услуги в сфере недропользования на территории Казахстана. Мы одна
+                из первых компаний являющихся лицензированным консультантом в
+                сфере минеральных ресурсов (лицензия №AFSA-A-LA-2020-0004).
+                Мinerals Operating зарегистрирована в Международном Финансовом
+                Центре «Астана» (МФЦА) и осуществляет деятельность, основанную на
+                нормах и принципах Английского общего права.
+              </p>
+            </div>
+  
+            <div class="main__wrapper-common-best">
+              <div class="main__wrapper-common-best-item">
```

Comment:
Вот эту штуку, на самом деле, можно было бы сделать и блоком (хотя и не обязательно).
```suggestion
              <div class="wrapper-common-best-item">
```